### DISCUSSION TITLE
Is it possible to disable individual elements of the base header?

### DISCUSSION THREAD

**Discussion by lhvy** [lowlighter/metrics#994] [Q&A] [answered]
I run CI every 24 hours to update my readme image, and I've noticed it's committing almost every day due to the fact that the calendar in the header of the base plugin (green and white squares in the attached screenshot) is updating and will update for a full 2 weeks after any activity.

<img width="483" alt="image" src="https://user-images.githubusercontent.com/40507205/163803593-41fdcaf7-d30d-4975-91c2-36bf7b0a1b15.png">

Is it possible through the config to disable this item in any way without removing the entire header?

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !
While there's no option to directly remove it, it should be doable using [`extras_js`](https://github.com/lowlighter/metrics/blob/master/source/plugins/core/README.md#%EF%B8%8F-custom-javascript-scripting) option and removing the div directly:
```yml
extras_js: |
  document.querySelector(".field.calendar")?.remove()
```
